### PR TITLE
Use id instead of _display to avoid 'SIA'

### PR DESCRIPTION
**Pull request by boris-arkenaar**
closes signalen/frontend#159

### CODE REVIEW

File: src/types/api/public-incident.ts [hunk 1]
```diff
@@ -6,6 +6,7 @@ enum State {
 }
 export interface PublicIncident {
   _display: string
+  id: number
```
Comment (kiruys) [inline src/types/api/public-incident.ts:9]: More straightforward if _display is removed all together then.

File: src/signals/incident/containers/IncidentReplyContainer/__tests__/IncidentReplyContainer.test.tsx [hunk 2]
```diff
@@ -47,7 +47,7 @@ describe('IncidentReplyContainer', () => {
       await waitFor(() => {
         screen.getByRole('heading', { name: 'Aanvullende informatie' })
         screen.getByRole('heading', { name: 'Uw melding' })
-        screen.getByText('Nummer: SIA-1234')
```
Comment (janjaap) [inline src/signals/incident/containers/IncidentReplyContainer/__tests__/IncidentReplyContainer.test.tsx:50]: I'm not sure if this particular format is used anywhere else in the whole process of incident management. Can we maybe move the `SIA-` prefix to the configuration?

Comment (boris-arkenaar) [reply src/signals/incident/containers/IncidentReplyContainer/__tests__/IncidentReplyContainer.test.tsx:50]: You mean, the `SIA-` prefix is used in other parts of the app, for Amsterdam, but configurable? And so it should be

Comment (janjaap) [reply src/signals/incident/containers/IncidentReplyContainer/__tests__/IncidentReplyContainer.test.tsx:50]: The prefix is indeed used in the administration of Amsterdam, so it needs to be configurable.

**Comment by janjaap** [comment]
@boris-arkenaar Can we close this PR and move the responsibility for providing the correct label to the backend?

**Comment by boris-arkenaar** [comment]
Yes, let's close it!

**Pull request closed without merging**